src/pages/SubmitWorkflow.js

```diff
@@ -11,14 +13,37 @@ export const SubmitWorkflow = () => {
   const [workflowInputs, setWorkflowInputs] = useState()
   const [showInputsPage, setShowInputsPage] = useState(false)
 
+  const signal = useCancellation()
+
+  const submitRun = async () => {
+    const runRes = await Ajax(signal).Cbas.submitRun(workflowUrl, workflowInputs)
+    console.log(runRes)
+  }
+
   return div([
     headerBar(),
     div({ style: { margin: '4rem' } }, [
       h2(['Submit a workflow']),
       div(['Submit your Terra workflows with the Cromwell engine. Full featured workflow submissions coming soon!']),
       div({ style: { marginTop: '2rem' } }, [
-        !showInputsPage && h(WorkflowSource, { workflowUrl, setWorkflowUrl, onClick: () => setShowInputsPage(true) }),
-        showInputsPage && h(WorkflowInputs, { workflowUrl, workflowInputs, setWorkflowInputs, displayWorkflowUrlPage: () => setShowInputsPage(false) })
+        !showInputsPage && h(WorkflowSource, { workflowUrl, setWorkflowUrl }),
+        !showInputsPage && div({ style: { display: 'flex', justifyContent: 'flex-end', marginTop: '1rem' } }, [
+          h(ButtonPrimary, {
+            disabled: !workflowUrl,
+            onClick: () => setShowInputsPage(true)
+          }, ['Use workflow'])
+        ]),
+        showInputsPage && h(WorkflowInputs, { workflowUrl, workflowInputs, setWorkflowInputs }),
+        showInputsPage && div({ style: { display: 'flex', marginTop: '1rem', justifyContent: 'flex-end' } }, [
+          h(ButtonOutline, {
+            onClick: () => setShowInputsPage(false)
+          }, ['Back to previous page']),
```

Comment:
Maybe `"Change selected workflow"`? Since it isn't really changing what page we're on?